src/development/tools/devtools/release-notes/release-notes-2.17.0-src.md

```diff
@@ -8,6 +8,12 @@ towards some new features that are coming soon!
 See the full list of commits for DevTools 2.17.0
```

Comment:
remove this section of general changes if there are none to note. This blurb was specific to the 2.16.0 release and shouldn't be included in all release notes